# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/res/debug_font.glsl
# Commit: 15da94c6a7b1
# Full Hash: 15da94c6a7b19f971d3dffaa4cab8edb8e41cb62
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-15 09:29:54
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r 72ddac486037 -r 15da94c6a7b1 gfx/wr/webrender/res/debug_font.glsl
--- a/gfx/wr/webrender/res/debug_font.glsl	Tue Dec 15 01:46:51 2020 +0200
+++ b/gfx/wr/webrender/res/debug_font.glsl	Mon Dec 14 21:48:41 2020 +0000
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#define WR_FEATURE_TEXTURE_2D
+
 #include shared,shared_other
 
 varying vec2 vColorTexCoord;
@@ -22,7 +24,7 @@
 
 #ifdef WR_FRAGMENT_SHADER
 void main(void) {
-    float alpha = texture(sColor0, vec3(vColorTexCoord.xy, 0.0)).r;
+    float alpha = texture(sColor0, vColorTexCoord).r;
     oFragColor = vColor * alpha;
 }
 #endif